system_v10/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "fil_actor_system_v10"
+description = "Builtin system actor for Filecoin"
+version = "10.0.0-alpha.1"
+license = "MIT OR Apache-2.0"
+authors = [
+  "ChainSafe Systems <info@chainsafe.io>",
+  "Protocol Labs",
+  "Filecoin Core Devs",
+]
+edition = "2018"
+repository = "https://github.com/filecoin-project/builtin-actors"
+keywords = ["filecoin", "web3", "wasm"]
+
+[dependencies]
+fil_actors_runtime_v10 = { workspace = true }
+fvm_shared = { version = "2.0.0-alpha.2", default-features = false }
```

Comment:
Do we not use the most recent version of fvm_shared (as in the current built-in actors repo): https://github.com/filecoin-project/builtin-actors/blob/a34f5c634dc82202ac1a9895a2b065dd41f44939/actors/system/Cargo.toml#L18

Reply:
Fixed